### DISCUSSION TITLE
vaultwarden_ldap password with argon2

### DISCUSSION THREAD

**Discussion by AlexisPiST** [dani-garcia/vaultwarden#3543] [Q&A]
Hello,

In Vaultwarden it is possible to encrypt the token with Argon2, but when I use the vaultwarden_ldap container it is not possible to use the Argon2 encrypted token, so it is a pain to leave it in the clear here.

With slappw-argon2 it is possible but it is compatible with vaultwarden_ldap ?

Has anyone found a solution to this problem?

**Comment by BlackDex** [comment]
I would suggest to ask that question there, but as far as i know `vaultwarden_ldap` is not maintained anymore.
Also, `vaultwarden_ldap` needs the password in plain-text, else it can't send the correct password to Vaultwarden.

**Reply by AlexisPiST** [reply]
Ok thanks for your quick response ! :)
